docs/style/entities/author.md

```diff
@@ -60,144 +60,156 @@ should be related to
 ## Author Credits
 This section explains about author credits.
 
-When entering books into BookBrainz, it's important to understand the difference between the concept of an <u>Edition</u>, the physical item you can hold in your hands, and the literary concept of <u>Works</u> that books contain. (see also [Wikipedia](https://en.wikipedia.org/wiki/Creative_work) for more information.)<br>
-{add an example of small collection of short stories by different author but only one author name on the cover}
+When entering books into BookBrainz, it's important to understand the difference between the concept of an <u>Edition</u>, the physical item you can hold in your hands, and the literary concept of <u>Works</u> that books contain. (see also [Wikipedia](https://en.wikipedia.org/wiki/Creative_work) for more information.)
+To illustrate the difference, see these two Editions which contain the same underlying Work, the novel [The Handmaid’s Tale](https://bookbrainz.org/work/f23d22e8-11ba-440a-a990-075152266c21): this [mass market paperback](https://bookbrainz.org/edition/a70f9eb1-b19b-4467-8f6b-c96c4616d829) published by [Seal Books](https://bookbrainz.org/publisher/e8b15024-7b26-4258-9d32-9367890b8a6d), and this [hardcover edition](https://bookbrainz.org/edition/c5d0a3e0-0eb6-4f07-b9f4-5d49a6819524) published by [Heinemann](https://bookbrainz.org/publisher/62754764-fcbd-469a-8ee1-579ccc7747b4).
```

Comment:
This could do with more explanations and less links to unrelated entities, after reading it again.

Reply:
Eventually you'll want a work page and an edition page probably, or at least one for the two combined - and then I'd link to that from here and drop all of this. The example does illustrate the difference fairly well IMO at least (although the links to the publishers are probably not needed - not sure they hurt either though).